cmd/di.go

```diff
@@ -509,3 +496,37 @@ func (di *Dependencies) bootstrapNATComponents(options node.Options) {
 		di.NATPinger = &traversal.NoopPinger{}
 	}
 }
+
+// FallbackResolverFactory creates a fallback resolver given the params
+func FallbackResolverFactory(ipResolver ip.Resolver, options node.OptionsLocation, configDirectory string) (location.Resolver, error) {
```

Comment:
if `options.Type == "manual"`, it will not create a fallback resolver, but rather a static one. Thus we should change the name to avoid confusion. _Factory_ is a noise word; we should name the function after what it does. WDYT about something similar to`CreateLocationResolver`?

Reply:
Remade type `manual` to be a fallback resolver